src/module-elasticsuite-core/Model/Search/Request/RelevanceConfig/Structure/Data.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Thomas Klein <thomasklein876@gmail.com>
+ * @copyright 2020 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+declare(strict_types = 1);
+
+namespace Smile\ElasticsuiteCore\Model\Search\Request\RelevanceConfig\Structure;
+
+use Magento\Framework\Config\Data\Scoped;
+
+/**
+ * Provide scoped configuration for the elasticsuite components
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCore
+ * @author   Thomas Klein <thomasklein876@gmail.com>
+ */
+class Data extends Scoped
+{
+    /**
+     * Merge default config data with extended settings
+     *
+     * @param array $config
```

Comment:
Almost there (sorry :|), could you add a comment after "$config" and have an empty comment line between [@param ](url)and `@return` ?
see for instance https://github.com/Smile-SA/elasticsuite/blob/67fbc17c2bfbf9383e654b454e50e8cbedf4643a/src/module-elasticsuite-core/Index/Mapping/Field.php#L278

Regards,